frontend/src/Context/WebSocketContext.jsx

```diff
@@ -9,7 +9,7 @@ export const WebSocketProvider = ({ children }) => {
     const [error, setError] = useState(null);
 
     useEffect(() => {
-        const ws = new WebSocket('wss://co2-api.duckdns.org');
+        const ws = new WebSocket('wss://third-year-studio-2024-fresh-air-backend.onrender.com');
```

Comment:
The render account is no longer active.   
This needs to be set to the duckdns url. Or (should) be moved to the env file when we run our compose.
Can be done at a later date, but for now it should be set to the live url